client/python/samples/get_grpc_model_status.py

```diff
@@ -24,20 +24,20 @@
                     dest='model_name')
 parser.add_argument('--model_version', default=0, type=int, help='Model version to query. Lists all versions if omitted',
                     dest='model_version')
+parser.add_argument('--timeout', default=10.0, help='Request timeout. default: 10.0',
+                    dest='timeout')
 args = vars(parser.parse_args())
 
 # configuration
 service_url = args.get('service_url')   # default='localhost:9000'
 model_name = args.get('model_name')  # default='resnet'
 model_version = args.get('model_version')   # default=0
+timeout = args.get('timeout')   # default=10.0
 
 # creating grpc client
 client = make_grpc_client(service_url)
 
-# creating status request
-request = make_grpc_status_request(model_name, model_version)
-
 # getting model status from the server
-status = client.get_model_status(request)
+status = client.get_model_status(model_name, model_version, timeout)
 status_dict = status.to_dict()
```

Comment:
Does it work? `get_model_status` calls `to_dict()` internally and should return dictionary